StandardLibrary/runtimes/go/ImplementationFromDafny-go/FileIO/externs.go

```diff
@@ -38,3 +58,37 @@ func (_static CompanionStruct_Default___) INTERNAL_WriteBytesToFile(path _dafny.
 	}
 	return false, _dafny.EmptySeq
 }
+
+func (_static CompanionStruct_Default___) INTERNAL_AppendBytesToFile(path _dafny.Sequence, bytes _dafny.Sequence) (isError bool, errorMsg _dafny.Sequence) {
+	p := func() string {
+		var s string
+		for i := _dafny.Iterate(path); ; {
+			val, notEndOfSequence := i()
+			if notEndOfSequence {
+				s = s + string(val.(_dafny.Char))
+			} else {
+				return s
+			}
+		}
+	}()
+
+	// Create directories
+	os.MkdirAll(filepath.Dir(p), os.ModePerm)
+
+	bytesArray := _dafny.ToByteArray(bytes)
+
+	f, err := os.OpenFile(p, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return true, _dafny.UnicodeSeqOfUtf8Bytes(err.Error())
```

Comment:
`_dafny.UnicodeSeqOfUtf8Bytes` might not be compatible with our implementation of dafny Strings. We expect dafny Go strings to be in `dafny.Char` while `UnicodeSeqOfUtf8Bytes` uses `CodePoint`. It's the same reason why `dafny.SequenceVerbatimString(path, false)` was failing.